### PR TITLE
Initial generator code.

### PR DESCRIPTION
**Pull request by DamianRodziewicz**
Resolves GH-26

### CODE REVIEW

**Comment by mateuszwyszynski** [comment]
LGTM :)

**Review by mateuszwyszynski — DISMISSED** [review]
LGTM :)

File: man/SpaDES.shiny-package.Rd [hunk 1]
```diff
@@ -11,7 +11,7 @@ The \pkg{SpaDES.shiny} package
 \seealso{
 Useful links:
 \itemize{
-  \item \url{https://predictiveecology.github.io/SpaDES.shiny/}
+  \item \url{http://spades.shiny.predictiveecology.org/}
```
Comment (mateuszwyszynski) [inline man/SpaDES.shiny-package.Rd]: Didn't we agree that we won't be applying this change (which is automatically done by Roxygen)? This new page is under development. The deleted one is operating.

**Review by mateuszwyszynski — CHANGES_REQUESTED** [review]
Maybe we could add whisker package to required packages?

Except that the code generates an app which is running :)

File: man/SpaDES.shiny-package.Rd (hunk 1 above)
Comment (DamianRodziewicz) [reply man/SpaDES.shiny-package.Rd]: Fixed

**Comment by DamianRodziewicz** [comment]
Comments addressed, ptal.

**Review by mateuszwyszynski — APPROVED** [review]

**Comment by mateuszwyszynski** [comment]
LGTM. Whisker does download automatically :)

File: man/SpaDES.shiny-package.Rd (hunk 1 above)
Comment (achubaty) [reply man/SpaDES.shiny-package.Rd]: the correct URL is http://spades-shiny.predictiveecology.org

**Pull request merged**